daemon/containerd/mount.go

```diff
@@ -0,0 +1,51 @@
+package containerd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/containerd/containerd/mount"
+	"github.com/docker/docker/container"
+	"github.com/docker/docker/pkg/containerfs"
+	"github.com/sirupsen/logrus"
+)
+
+// Mount mounts and sets container base filesystem
+func (i *ImageService) Mount(ctx context.Context, container *container.Container) (err error) {
+	snapshotter := i.client.SnapshotService(i.snapshotter)
+	mounts, err := snapshotter.Mounts(ctx, container.ID)
+	if err != nil {
+		return err
+	}
+
+	tempMountLocation := os.TempDir()
+
+	root, uerr := os.MkdirTemp(tempMountLocation, "containerd-mount")
```

Comment:
Not sure why it's called `uerr`?
```suggestion
	root, err := os.MkdirTemp(tempMountLocation, "containerd-mount")
```

Reply:
Maybe we should put `"rootfs-mount"` as the pattern to `mkdirtemp`?

Reply:
Done